flakes/grep.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
```

Comment:
We could add `set -x` to each bash script to print each executed command.

Reply:
I'm not a fan of `set -x` in scripts that has it's own output (I know it's only stderr), as it's noisy and might be confusing. Perhaps just let's jus `set -x` in `./run-tests.sh`?

Reply:
I've tried doing `set -x` in run-tests and it's not really readable, so think I won't do it for now.